### PR TITLE
docs: update the latest faq info

### PR DESCRIPTION
**Pull request by beryl678**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/nightly/en/faq-and-others/faq.md [hunk 1]
```diff
@@ -11,23 +11,23 @@ Common use cases for time-series database include but are not limited to the fol
 
 ## How is GreptimeDB's performance compared to other solutions?
 
-GreptimeDB has released v0.7, with functionalities set to improve progressively. For detailed TSBS test results, refer to the link below: https://github.com/GreptimeTeam/greptimedb/blob/main/docs/benchmarks/tsbs/v0.7.0.md.
+GreptimeDB is still in its early stage and under rapid iterations. We are still optimizing our performance and enriching the features. Performance benchmark will be posted on our website as soon as it's ready. Please stay tuned.
```
Comment (nicecui) [inline docs/nightly/en/faq-and-others/faq.md:14]: It seems this is the older answer of this question

File: docs/nightly/en/faq-and-others/faq.md [hunk 2]
```diff
@@ -11,23 +11,23 @@ Common use cases for time-series database include but are not limited to the fol
...
 
 ## How is the performance of GreptimeDB when used for non-time-series DB tables?
 
 GreptimeDB supports SQL and can deal with non-time-series data, especially efficient for high concurrent and throughput data writing. However, we develop GreptimeDB for a specific domain (time-series scenarios), and it doesn't support transactions and can't delete data efficiently.
 
-## Does GreptimeDB have a Go driver?
+## Does GreptimeDB have a Golang driver?
```
Comment (nicecui) [inline docs/nightly/en/faq-and-others/faq.md:20]: same as above

File: docs/nightly/en/faq-and-others/faq.md [hunk 3]
```diff
@@ -81,41 +79,135 @@ Please refer to [features that you concern](/user-guide/concepts/features-that-y
...
+
+GreptimeDB's data is distributed in timestamp order, so there is no need to additionally shard/partition by timestamp. It is recommended to shard by primary key instead.
+
+## How many servers are generally needed to set up a reliable GreptimeDB cluster, and how should Frontend, Datanode, and Metasrv be deployed? Should each node run all three services regardless of the number of nodes?
+
+A minimum of 3 nodes is required, with each node running the 3 services: metasrv, frontend, and datanode. However, the exact number of nodes depends on the scale of data being handled.
+
+It is not necessary to deploy all three services on each node. A small-sized cluster can be set up with 3 nodes dedicated to metasrv. Frontend and datanode can be deployed on equal nodes, with one container running two processes.
+
+## Several questions about GreptimeDB: Does GreptimeDB v0.7 support inverted indexes, and does it use Tantivy? In v0.8, does the Flow Engine (pre-computation) feature support PromQL syntax for calculations? Will Metasrv support storage backends like MySQL or PostgreSQL?
```
Comment (nicecui) [inline docs/nightly/en/faq-and-others/faq.md:201]: It is better to split this question to several questions.

File: docs/nightly/en/faq-and-others/faq.md [hunk 4]
```diff
@@ -81,41 +79,135 @@ Please refer to [features that you concern](/user-guide/concepts/features-that-y
...
+
+Yes, we have tested Tantivy, and it meets our current requirements very well.
+
+This is a good suggestion. Currently, the Flow Engine does not support PromQL syntax for calculations. We will evaluate this, as it seems theoretically feasible.
+
+We have developed an abstraction layer for Metasrv, but it does not yet support RDBMS backends. Support for MySQL and PostgreSQL is planned.
+
+For further suggestions, please open an issue on our GitHub repository.
+
+## What is the best way to downsample interface traffic rates (maximum rate within every hour) from multiple NICs across thousands of computers every 30 seconds, so that the data can be kept for many years?
```
Comment (nicecui) [inline docs/nightly/en/faq-and-others/faq.md:211]: please write the words for the NIC abbreviation

**Review by nicecui — APPROVED** [review]

File: docs/nightly/en/faq-and-others/faq.md [hunk 5]
```diff
@@ -83,10 +81,9 @@ Please refer to [features that you concern](/user-guide/concepts/features-that-y
 
 Currently, GreptimeDB's compatibility efforts are primarily focused on the implementation of native PromQL. Going forward, we will continue to enhance compatibility with MetricQL's extended syntax.
 
-## Should I use the command "drop database" to delete a database?
+## If I delete the database, can I use the `drop database` command?
 
-Yes, you can use `DROP DATABASE db_name` command to delete a database.
-Please refer to the [DROP](/reference/sql/drop.md#drop-database) document for more details.
+Yes, the `drop database` command has been implemented in version 0.8. You can refer to the official documentation for usage: [`Drop Database`](https://docs.greptime.com/reference/sql/drop#drop).
```
Comment (nicecui) [inline docs/nightly/en/faq-and-others/faq.md:86]:
```suggestion
Yes, the `DROP DATABASE` command has been implemented in version 0.8. You can refer to the official documentation for usage: [`Drop Database`](https://docs.greptime.com/reference/sql/drop#drop).
```

File: docs/nightly/en/faq-and-others/faq.md [hunk 6]
```diff
@@ -83,10 +81,9 @@ Please refer to [features that you concern](/user-guide/concepts/features-that-y
 
 Currently, GreptimeDB's compatibility efforts are primarily focused on the implementation of native PromQL. Going forward, we will continue to enhance compatibility with MetricQL's extended syntax.
 
-## Should I use the command "drop database" to delete a database?
+## If I delete the database, can I use the `drop database` command?
```
Comment (nicecui) [inline docs/nightly/en/faq-and-others/faq.md:84]:
```suggestion
## If I delete the database, can I use the `DROP DATABASE` command?
```

File: docs/v0.8/en/faq-and-others/faq.md [hunk 7]
```diff
@@ -83,10 +81,9 @@ Please refer to [features that you concern](/user-guide/concepts/features-that-y
 
 Currently, GreptimeDB's compatibility efforts are primarily focused on the implementation of native PromQL. Going forward, we will continue to enhance compatibility with MetricQL's extended syntax.
 
-## Should I use the command "drop database" to delete a database?
+## If I delete the database, can I use the `drop database` command?
```
Comment (nicecui) [inline docs/v0.8/en/faq-and-others/faq.md:84]:
```suggestion
## If I delete the database, can I use the `DROP DATABASE` command?
```

File: docs/v0.8/en/faq-and-others/faq.md [hunk 8]
```diff
@@ -83,10 +81,9 @@ Please refer to [features that you concern](/user-guide/concepts/features-that-y
...
 
-Yes, you can use `DROP DATABASE db_name` command to delete a database.
-Please refer to the [DROP](/reference/sql/drop.md#drop-database) document for more details.
+Yes, the `drop database` command has been implemented in version 0.8. You can refer to the official documentation for usage: [`Drop Database`](https://docs.greptime.com/reference/sql/drop#drop).
```
Comment (nicecui) [inline docs/v0.8/en/faq-and-others/faq.md:86]:
```suggestion
Yes, the `DROP DATABASE` command has been implemented in version 0.8. You can refer to the official documentation for usage: [`Drop Database`](https://docs.greptime.com/reference/sql/drop#drop).
```

**Review by tisonkun — APPROVED** [review]

**Pull request merged**